prow/scripts/cluster-integration/busola-ui-tests-k3s.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+LOCAL_KYMA_DIR="./local-kyma"
+K3S_DOMAIN="local.kyma.dev"
+CYPRESS_IMAGE="eu.gcr.io/kyma-project/external/cypress/included:7.3.0"
+
+# pull the large image in the background to save some time
+# docker pull --quiet $CYPRESS_IMAGE &
+
+prepare_k3s() {
+    pushd ${LOCAL_KYMA_DIR}
+    ./create-cluster-k3s.sh
+    echo "k3s cluster created √"
+    kubectl cluster-info
+    popd
+}
+
+generate_cert(){
```

Comment:
Maybe it would be good to ad an `echo` at the beginning of every step so if it gets stuck for example to know when it stuck exactly?